scripts/callback.js

```diff
@@ -38,15 +49,9 @@ async function index(req, res) {
  * @param {string} baseUrl
  * @param {string} code
  */
-async function fetchToken(baseUrl, code) {
+async function fetchToken(sgidService, code) {
```

Comment:
Oops, didn't catch this earlier. Why not use `sgidService`'s methods directly instead on relying on these functions?

Reply:
RIght, because the previous code was structured this way, I thought it was good practice to separate the methods that call sgidService and isolate them in another file. I think what you suggested is a cleaner way to structure the code, will incorporate the changes.